GoogleCast/BasicCastPlayer/swift/BasicCastPlayer.xcodeproj/project.pbxproj

```diff
@@ -51,11 +51,19 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		96A56DA5CA88420DAF76B31D /* Pods */ = {
```

Comment:
Oops! Let's get this removed.

Reply:
Looks like there are a few instances of this, let's get them all cleaned up

Reply:
Nice catch. After `pod install`, there are project modifications - new file groups get added and new build phases get added.

Reply:
Running `pod deintegrate` (for each project) should take care of these easy peasy.